.github/workflows/README.md

```diff
@@ -928,6 +928,12 @@ This workflow leverages your Terraform project to provision GCP resources.
 
 Create a service account and save its key as a GitHub secret so you can provide it to the variable `google-credentials`.
 
+### Dependencies
+
+This workflow is built from multiple composite actions listed below:
+
+- [action-github-changelog-generator](https://github.com/heinrichreimer/action-github-changelog-generator)
```

Comment:
Again I am a bit worried about this action.
https://github.com/heinrichreimer/action-github-changelog-generator/issues/42
Take the issue as an example, the issue for the depricated `set-ouput` command was opened but never fixed.

Reply:
@MichaelKora is in the process of testing the other ones

Reply:
am closing this resolving this conversion since is covered by the [previous](https://github.com/bakdata/ci-templates/pull/84#discussion_r1146085648) one.